osu.Game.Rulesets.Mania/UI/DefaultColumn.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.UI.Scrolling;
+using osu.Game.Rulesets.Mania.Objects.Drawables;
+using osu.Game.Rulesets.Mania.UI.Components;
+
+namespace osu.Game.Rulesets.Mania.UI
+{
+    public class DefaultColumn : Column, IHasAccentColour
+    {
+        private readonly ColumnBackground background;
+        private readonly ColumnKeyArea keyArea;
+        private readonly ColumnHitObjectArea hitObjectArea;
+
+        // Pull up more stuff when needed.
+        protected override Container HitTargetContainer { get; }
```

Comment:
The column should have some basic layout associated with it, such as the positioning of the hit target and associated containers.

I'd prefer that the shared structure (hit target + children, key area, and top-level) were moved into the base class. The `DefaultColumn` would then inject its background wherever it needs to go.